exemples/admin/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "example",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "watch": "rm -rf build && mkdir build && tsc -p src -w",
+    "build": "rm -rf build && mkdir build && tsc -p src || true",
+    "start": "http-server -c-1 ."
+  },
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "typescript": "^1.8.10"
+  },
+  "dependencies": {
+    "@angular": "^2.0.0-rc.1",
+    "hm-admin": "0.1.0",
+    "es6-shim": "^0.35.0",
+    "http-server": "^0.8.5",
+    "reflect-metadata": "0.1.3",
+    "systemjs": "0.19.26"
+  }
+}
```

Comment:
The exemple is à part of this project. It allows dev to test visually the component. So the package.json is the same. But you can save all dependencies of exemple in devDependencies IMO